includes/blocks/example-block/block.json

```diff
@@ -0,0 +1,13 @@
+{
+	"title": "Example Block",
+	"description": "An Example Block",
+	"text-domain": "tenup-scaffold",
+	"name": "tenup/example",
+	"icon": "feedback",
```

Comment:
@ryanwelcher on projects recently we've been adding `example` / `preview` properties to show a preview of the block before adding it to the editor. This could either be a relative URL to an image or a server-side rendered preview of the block. Would be nice to have a way to handle that!

Reply:
Good point. This block is really only intended to provide context/direction when creating one so we should probably have a more complete example of the` block.json` file

Reply:
Agreed - happy with moving forward however you see fit.